### DISCUSSION TITLE
Alpine

### DISCUSSION THREAD

**Discussion by nallej** [louislam/dockge#456] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

Installation package for Alpine
Any plan to make Dockge available on Alpine Linux. As Alpine is one of the most popular for building servers, I would like to see Dockge on Alpine too.

### ✔️ Solution

Make a package for the community apk.

### ❓ Alternatives

I will use Portainer for now.

**Comment by RavHilton** [comment]
Dockge runs as a container, so it works perfectly fine under Docker on Alpine Linux already.
